### ISSUE TITLE
Decision Proposal 058 - Payees v1

### ISSUE THREAD

**Issue by JamesMBligh** [ConsumerDataStandardsAustralia/standards#58]
This issue has been opened to propose and publish the final decision paper for Payees that establishes version 1 of these end points and payloads.

The proposed decision is below.  Feedback will close on 26th April.
[Decision 058 - Payees V1.pdf](https://github.com/ConsumerDataStandardsAustralia/standards/files/3061946/Decision.058.-.Payees.V1.pdf)

**Comment by JamesMBligh** [comment]
Please note that the decision proposal for this item has been added to the issue description

-JB-

**Comment by NationalAustraliaBank** [comment]
NAB would like to provide the following initial feedback on the Payees v1 definition

#### Security scopes
The security scopes in the decision currently reflect the _bank scheduled payments and debit_ scopes.
We expect it to be `bank_basic_accounts` for `GET /banking/payees` and `bank_payees` for `GET /banking/payees/{payeeId}`.

#### `payId` object
The `name` field is now mandatory, whereas in the current Swagger definition it is currently optional. What was the reason for this change?
As the PayID name must be resolved using a centralised and external service during the time of payment creation, data holders may not store the name when saving payees.

**Comment by anzbankau** [comment]
Hi James

We have the following points on the Payee endpoints:

**International.beneficiary.country**
We don’t capture this as a value for international payees (we have the country for the recipient bank, but not the actual beneficiary).

**Security scopes**
The decision proposal states the scopes being: Bank Scheduled Payments And Debits
However the draft version on github has the account basic scope (for payee list) and payee scope (for payee detail). This seems more appropriate - or alternatively the payee scopes could be used for both the list and detail.

**Domestic.payeeId.name**
This field must be optional - the customer may have created the payee but never made a payment to it, in which case the “The name assigned to the PayID by the owner of the PayID” won’t be available as under NPP rules the PayID can only be resolved during the making of a payment.

**Biller**
The biller object is set as mandatory, but should be conditional.


**Domestic.account**
This description is missing statement about when it is conditionally mandatory (description is empty)

**Biller.crn**
The field is conditionally mandatory, but doesn't have a description to state when it is required.

**International**
The international object is set as mandatory, but should be conditional.

**BeneficiaryDetails.message** 
The description states:
Response message for the payment
'Response' doesn't seem appropriate if it is an intended message from the payee.

**Comment by anzbankau** [comment]
Hi James,

One more item for the Payee end point:

**International.bankDetails.country**
Is this meant to be the country of the recipient branch, or country where the recipient bank in incorporated? If it is the country of the bank - this should be changed to optional as this isn’t available (we have the country for the recipient branch).

**Comment by JamesMBligh** [comment]
Responses to feedback are:

- The security scope for Payees was changed to match findings from the CX stream.  CS testing indicated that customers felt that payees, scheduled payments and direct debit authorisations were conceptually linked and preferred that consent be provided to these as a cluster rather than separately.  The decision has been updated accordingly.
- The name (for PayID) should be optional.  This has been corrected in swagger and the document.
- As the bank country is mandatory the beneficiary country has been made optional.  Also the bank country has been modified to indicate it is the country of the bank branch.
- biller and international objects have been corrected to conditional in the document.
- Updated the description of the UType conditional objects in the domestic objects.
- The crn field has been set to optional.  The conditional status appears to be a hangover from an earlier position related to the handling of Credit Card numbers.

I have updated the document in the main issues comment and have made but not yet published the swagger changes.  This will be done once the rest of the feedback from closed issues has been incorporated.

-JB-
